imports/ui/body.js

```diff
@@ -3,11 +3,11 @@ import './body.html'
 import { Template } from 'meteor/templating';
 import { Vendors } from '../api/userdata.js';
 
-Template.body.onCreated(function bodyOnCreated() {
+Template.register.onCreated(function bodyOnCreated() {
     console.log('Body created')
```

Comment:
yes u can change it to that. no template 'body' exists now. the content was included in other template 'register' in the last pull request. hence 'body' replace by 'register'.